analyze-question-sentiment.go

```diff
@@ -131,6 +132,26 @@ func analyzeDir(baseDir string, tag string, fromDate time.Time, toDate time.Time
 	return tr
 }
 
+// Discover the names of the subfolders inside dir (non-recursively).
+func readFolderNames(dirpath string) ([]string, error) {
+	dir, err := os.Open(dirpath)
+	if err != nil {
+		return nil, err
+	}
+	defer dir.Close()
+	entries, err := dir.ReadDir(0)
```

Comment:
Could you use `os.ReadDir` directly here?

Reply:
TIL, thanks